src/main/java/org/sagebionetworks/web/client/FeatureFlagKey.java

```diff
@@ -37,6 +37,9 @@ public enum FeatureFlagKey {
   // If enabled, use the re-implemented ACL Editor for entities
   REACT_ENTITY_ACL_EDITOR("REACT_ENTITY_ACL_EDITOR"),
 
+  // If enabled, sharing settings will appear in a dialog immediately after uploading one or more files.
+  SHOW_SHARING_SETTINGS_AFTER_UPLOAD("SHOW_SHARING_SETTINGS_AFTER_UPLOAD"),
```

Comment:
I'm not planning on adding this to the stack builder for now, since we don't need to dynamically toggle it. This is basically in experimental mode only, but it's easy to find + remove all of the toggles for this feature when we need to.